docs/reference/webhook.mdx

```diff
@@ -20,7 +20,7 @@ you can filter to just the events you are interested in subscribing to based on
 {
   "event": "experiment.metric.updated",
   "data": {
-    "experiment_id": "<experiment_id>",
+    "experiment_id": <experiment_id>,
```

Comment:
This is no longer valid JSON, but I wanted to make it clear to not expect a string representation of a numeric id.